### DISCUSSION TITLE
MacCatalyst with macOS Monterey and Xcode 14.0.1

### DISCUSSION THREAD

**Discussion by leonschoi** [unoplatform/uno#14905] [Q&A] [answered]
I am trying to use Uno in macOS Monterey (v.12.7.2) which has Xcode 14.0.1.  When I run uno-check, it looks for Xcode 15.0.1 only, and when I try to run a sample app in MacCatalyst, it gives an error message:

/usr/local/share/dotnet/packs/Microsoft.MacCatalyst.Sdk/17.0.8478/tools/msbuild/iOS/Xamarin.Shared.targets(1837,3): error : Could not find a valid Xcode app bundle at '/Library/Developer/CommandLineTools'

Is there a way to use Xcode 14.0.1 to run Uno in Monterey?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question. This would be a .NET SDK behavior that can be fixed by looking at the steps in https://github.com/dotnet/maui/issues/11339.
